pip-tools.in

```diff
@@ -0,0 +1,2 @@
+
+pip-tools >= 7 # .pip-tools.toml was introduced in v7
```

Comment:
Hmm, I'm not sure where I got `v7` from then...

Reply:
I think v7 introduces some breaking change since we're trying to stick to SemVer there. Maybe CLI option default swapped or something. But reading settings from configs alone wouldn't qualify for a major bump. But yes, it there were probably several changes going in at that time, one of which included adding support for configs and the following/breaking was released soon after. Looking at the changelog, I see that this was related to switching the depresolver to backtracking by default and dropping support for Python 3.7.